lib/puppet/indirector/facts/yaml.rb

```diff
@@ -5,10 +5,6 @@ class Puppet::Node::Facts::Yaml < Puppet::Indirector::Yaml
   desc "Store client facts as flat files, serialized using YAML, or
     return deserialized facts from disk."
 
-  def allow_remote_requests?
-    false
-  end
-
```

Comment:
I also verified yaml is still rejected due to the other checks we have in place, so 👍